ibm/mas_devops/roles/suite_app_config/vars/predict.yml

```diff
@@ -5,3 +5,17 @@ mas_app_ws_kind: PredictWorkspace
 
 mas_app_cfg_delay: 120
 mas_app_cfg_retries: 30
+
+mas_app_settings_wml_instance_id: "{{ lookup('env', 'CPD_WML_INSTANCE_ID') | default('openshift', true) }}"
+mas_app_settings_wml_url: "{{ lookup('env', 'CPD_WML_URL') | default('https://internal-nginx-svc.ibm-cpd.svc:12443', true) }}"
```

Comment:
Are these not the same as what is defined in roles/suite_app_config/defaults/main.yml ?

Reply:
will fix this